content/release-notes/2023-04-02.md

```diff
@@ -0,0 +1,7 @@
+---
+label: 'Plans'
+---
+
+### What's new
+
+- Free Tier: The _compute active time_ limit of 100 hours per month for non-primary branches, introduced on February 6, 2023, now applies to all Neon Free Tier projects. The limit was not applied to projects created before February 6, 2023 to give users time to adjust their usage or upgrade to a [paid plan](https://neon.tech/pricing). For information about Free Tier limits, see [Free Tier](https://neon.tech/docs/introduction/technical-preview-free-tier). If you have questions about this change or upgrading to a paid plan, please contact [Sales](https://neon.tech/contact-sales).
```

Comment:
I think it's February 24, and not 6. Or am I missing something?

Reply:
Hi @nikitakalyanov This looks like the release that introduced the 100 hours limit: https://github.com/neondatabase/cloud/pull/3795 
Did we actually implement the 100 hour limit on Feb 2nd? If you can confirm, I will update the dates.

Reply:
Looks like the merge date for that release was Feb 6

Reply:
Yeah, but this merge date corresponds to the release to staging. The release to production happened on 24 of February. We also discussed it here: https://neondb.slack.com/archives/C04382GUYCC/p1678799568986299?thread_ts=1678792974.993889&cid=C04382GUYCC

Reply:
Thank you. I will update here and a coupe of other places.